src/build/__init__.py

```diff
@@ -133,10 +133,11 @@ def __init__(self, srcdir='.', config_settings=None):  # type: (str, Optional[Co
 
         self._backend = self._build_system['build-backend']
 
-        try:
-            importlib.import_module(self._backend.split(':')[0])
-        except ImportError:  # can't mock importlib.import_module  # pragma: no cover
-            raise BuildException("Backend '{}' is not available".format(self._backend))
+        if 'backend-path' not in self._build_system:
+            try:
+                importlib.import_module(self._backend.split(':')[0])
```

Comment:
Arguably you could just do ``self._backend[self._backend.find(':')]`` 🤔 Which is more in spirit of what you want 🤔

Reply:
What if there is no `:`? `find` will return `-1`, so `self._backend[-1]`, which will return the last character of the backend string :grin:. `split` will always return a list, if it doesn't find anything, it will return a list with the original string.